docs/rackhd/event_notification.rst

```diff
@@ -1,105 +1,272 @@
+|              |                        | sku.unassigned         | information    | Triggered when node's `sku`       |
+|              |                        |                        |                | field is unassigned.              |
+|              |                        +------------------------+----------------+-----------------------------------+
+|              |                        | sku.updated            | information    | Triggered when node's `sku`       |
+|              |                        |                        |                | field is updated.                 |
+|              |                        +------------------------+----------------+-----------------------------------+
+|              |                        | obms.assigned          | information    | Triggered when node's `obms`      |
+|              |                        |                        |                | field is assigned.                |
+|              |                        +------------------------+----------------+-----------------------------------+
+|              |                        | obms.unassigned        | information    | Triggered when node's `obms`      |
+|              |                        |                        |                | field is unassigned.              |
+|              |                        +------------------------+----------------+-----------------------------------+
+|              |                        | obms.updated           | information    | Triggered when node's `obms`      |
+|              |                        |                        |                | field is updated.                 |
+|              |                        +------------------------+----------------+-----------------------------------+
+|              |                        | accessible             | information    | Triggered when node telemetry     |
+|              |                        |                        |                | OBM service (IPMI or SNMP) is     |
+|              |                        |                        |                | accessible                        |
+|              |                        |                        |                |                                   |
+|              |                        +------------------------+----------------+-----------------------------------+
+|              |                        | inaccessible           | information    | Triggered when node telemetry     |
+|              |                        |                        |                | OBM service (IPMI or SNMP) is     |
+|              |                        |                        |                | inaccessible                      |
++--------------+------------------------+------------------------+----------------+-----------------------------------+
```

Comment:
might consider break this big table into several small ones when more events are added in the future. i am ok to leave it as it is now